### PR TITLE
Fix code examples to allow easy copy/ paste

### PR DESCRIPTION
**Pull request by amitaibu**
Reactivity page was showing a few examples that were not correct, or not easily copied.

In the PR:
* Install all needed packages.
* Fix code examples mostly by fixing the `import`
* Added a fibonacci function I've copied from @johnpmayer
* Added `main`when it was missing, directing users to open the developer console

(on a side note, as an elm newbie the HTTP errors examples are still not very clear - would love more clarification/ more detailed examples)

### CODE REVIEW

**Comment by evancz** [comment]
This looks great, thank you!

My minor nitpicks would be:

  1. Do you mind making sure the imports are in alphabetical order? Currently `Graphics` often appears after `Task` and `Time` which is weird to me.
  2. Do you mind using the following definition of fibonacci?

```elm
fibonacci : Int -> Int
fibonacci n =
  if n <= 2 then
    1
  else
    fibonacci (n-1) + fibonacci (n-2)
```

It works better for bad input.

**Comment by amitaibu** [comment]
> My minor nitpicks would be ..

Fixed and rebased.

**Comment by evancz** [comment]
Great, thank you! It should be live on the site in a few minutes!

**Comment by amitaibu** [comment]
I think the site wasn't updated yet.

**Comment by evancz** [comment]
Ah, I forgot. I am trying now and there are actually some issues that are stopping it from compiling. I am trying to figure it out now.

File: src/guide/chapters/reactivity.md [hunk 1]
```diff
@@ -380,12 +396,32 @@ that chains a bunch of tasks together to measure how long it takes to evaluate
...
+
 getDuration : Task x Time
 getDuration =
   getCurrentTime
-    `andThen` \\start -> succeed (fibonacci 20)
-    `andThen` \\fib -> getCurrentTime
-    `andThen` \\end -> succeed (end - start)
+    `andThen` \start -> succeed (fibonacci 20)
+    `andThen` \fib -> getCurrentTime
+    `andThen` \end -> succeed (end - start)
```
Comment (evancz) [inline src/guide/chapters/reactivity.md]: Ah, here it is, you took off the escaping of the lambdas!

**Comment by evancz** [comment]
Okay, should be all set now. I missed the `\` issue in review. It's a super tricky reason that it needs to be double and I missed it in review too, so don't worry about it too much, not a huge deal! In the future though, try to keep your changes more minimal. Changing that was not necessary to achieve the goal of the PR, so it should be separate.

**Pull request merged**